notifier/github/notify.go

```diff
@@ -22,6 +22,16 @@ func (g *NotifyService) Notify(body string) (exit int, err error) {
 		return result.ExitCode, result.Error
 	}
 
+	_, isPlan := parser.(*terraform.PlanParser)
+	if isPlan {
+		if result.HasDestroy && cfg.WarnDestroy {
+			// Notify destroy warning as a new comment before normal plan result
+			if err = g.notifyDestoryWarning(body, result); err != nil {
+				return result.ExitCode, err
```

Comment:
This error is not important compared with "plan" result. 

If sending "plan" messaged is failed, it should return non-zero value here but this kinda failures which is not related to the main logic can be discard (or just logging).


What do you think?

Reply:
I see. Just flushing error log seems better.

Reply:
LGTM

Reply:
@dtan4 

Let's modify this later!